packages/block-library/src/post-breadcrumbs/edit.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import {
+	ToggleControl,
+	TextControl,
+	__experimentalToolsPanel as ToolsPanel,
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { RawHTML } from '@wordpress/element';
+import { useServerSideRender } from '@wordpress/server-side-render';
+
+/**
+ * Internal dependencies
+ */
+import { useToolsPanelDropdownMenuProps } from '../utils/hooks';
+const separatorDefaultValue = '/';
+
+export default function PostBreadcrumbEdit( {
+	attributes,
+	setAttributes,
+	context: { postId, postType },
+} ) {
+	const { separator, showHomeLink } = attributes;
+	const postTypeObject = useSelect(
+		( select ) => {
+			if ( ! postType ) {
+				return null;
+			}
+			return select( coreStore ).getPostType( postType );
+		},
+		[ postType ]
+	);
+	const blockProps = useBlockProps();
+	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
+	const { content } = useServerSideRender( {
+		attributes,
+		skipBlockSupportAttributes: true,
+		block: 'core/post-breadcrumbs',
+		urlQueryArgs: { post_id: postId },
+	} );
+	let placeholder = null;
+	// If no post context or the post type is not hierarchical, show placeholder.
+	// This is fragile because this block is server side rendered and we'll have to
+	// update the placeholder html if the server side rendering output changes.
+	if ( ! postId || ! postType || ! postTypeObject?.hierarchical ) {
+		const placeholderItems = [
+			showHomeLink && __( 'Home' ),
+			__( 'Ancestor' ),
+			__( 'Parent' ),
+		].filter( Boolean );
+		placeholder = (
+			<nav
+				style={ {
+					'--separator': `'${ separator }'`,
+				} }
+				inert="true"
+			>
+				<ol>
+					{ placeholderItems.map( ( text, index ) => (
+						<li key={ index }>
+							<a href={ `#${ text.toLowerCase() }` }>{ text }</a>
```

Comment:
```suggestion
							<a
								href={ `#post-breadcrumbs-pseudo-link-${ index }` }
							>
```

The text can be localized, so an incorrect href value may be applied.